examples/with-typescript-styled-components/package.json

```diff
@@ -8,15 +8,15 @@
   },
   "dependencies": {
     "next": "latest",
-    "react": "16.8.6",
-    "react-dom": "16.8.6",
-    "styled-components": "^4.1.3"
+    "react": "^16.8.0",
+    "react-dom": "^16.8.0",
```

Comment:
This is not required, better to keep the up to date versions.

Reply:
I've updated to the up to date version. Is this ok?